pinot-core/src/test/java/org/apache/pinot/core/data/manager/realtime/LLRealtimeSegmentDataManagerTest.java

```diff
@@ -63,23 +68,21 @@
 
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.ArgumentMatchers.anyInt;
-import static org.mockito.Mockito.anyString;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.*;
 
 
 // TODO Re-write this test using the stream abstraction
 public class LLRealtimeSegmentDataManagerTest {
   private static final String SEGMENT_DIR = "/tmp/" + LLRealtimeSegmentDataManagerTest.class.getSimpleName();
   private static final File SEGMENT_DIR_FILE = new File(SEGMENT_DIR);
   private static final String TABLE_NAME = "Coffee";
-  private static final int PARTITION_GROUP_ID = 13;
+  private static final int PARTITION_GROUP_ID = 0;
```

Comment:
What's the reason of changing `PARTITION_GROUP_ID` and `START_OFFSET_VALUE` in this test? We should be able to handle any partition and any offset

Reply:
Seems this is because we only create fake data for 2 partitions, so there is no data available for partition 13